### ISSUE TITLE
Missing info in section: Migration from the previous model to the carrier trunk

### ISSUE THREAD

**Issue by nfon-christophe** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#2882]
We're testing the migration from the previous to the new carrier trunk model and have found that there's a missing part in the documentation.

We get an error message upon attempting to run Remove-CsOnlinePSTNGateway :

Can not remove 'OnlinePSTNGateway', linked 'OnlineVoiceRoute' exists. 
Use 'Remove-CsOnlineVoiceRoute'

Can the documentation be clarified? It's necessary to wait a few hours before attempting to remove the PSTN Online Gateway

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1cb0c2ef-6274-9167-ecc6-3695d7460a8d
* Version Independent ID: aacf7318-02fc-a1c3-369c-65e6936fdb3c
* Content: [Configure a Session Border Controller for multiple tenants](https://docs.microsoft.com/en-us/MicrosoftTeams/direct-routing-sbc-multiple-tenants#feedback)
* Content Source: [Teams/direct-routing-sbc-multiple-tenants.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/direct-routing-sbc-multiple-tenants.md)
* Service: **msteams**
* GitHub Login: @CarolynRowe
* Microsoft Alias: **crowe**

**Comment by lindspea** [comment]
@nfon-christophe Thank you for reaching out and helping improve MS Docs.  Are you suggesting adding a note to mention that it's important to wait a few hours before attempting to remove the PSTN Online Gateway in the article?

**Comment by nfon-christophe** [comment]
@lindspea Yes I was. When we originally tested the change on our dev environment, we added the PSTN Online Gateway to the carrier tenant, the tested that direct routing for customer tenant was still working. That took circa 45 minutes. We then proceeded to remove the PSTN Online Gateway from the customer tenant and received an error message stating that the gateway was still used in an online voice route. As the time required for sync to fully complete seems to be very different depending on the changes being made, it would be useful to have some guidance from Microsoft on how long we should wait before proceeding to the next stage
